adguardhome.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "adguardhome",
+    "release": "12.2-RELEASE",
+    "artifact": "https://github.com/leandroscardua/iocage-plugin-adguardhome.git",
+    "official": false,
+    "pkgs": [
+        "adguardhome"
+    ],
+    "properties": {
+        "dhcp": 1,
```

Comment:
Do you really need `dhcp` here or could it be changed to `NAT` instead? I do like DHCP for all my jails and use it for all of them but defaulting to `dhcp` without any `nat` forwards set can cause issues like: https://github.com/ix-plugin-hub/iocage-plugin-index/issues/236

Reply:
As the AdGuardHome works filter DNS query, using it as nat, I`m afraid to overloded the Truenas main IP and avoid conflict with other pluings like Bind DNS and Dnsmasq already on the ecosystem.
by default, the Adguardhome uses port 53, to receive the dns query, furthermore if the user changes that port to anything else and break the access to DNS.

yes, I can change it something like that? or not? or no need to the webpart?

        "nat": 1,
        "nat_forwards": "tcp(80:80)"    ## Web UI
        "nat_forwards": "tcp(53:53)"   ## DNS Query

Reply:
That could "work" (just the `nat_forwards` property should be specified only once, if more mappings are needed they should be comma separated inside the value so something like: `"nat_forwards": "tcp(80:8585),tcp(53:53)"`. 

Although, like you said before this will collide with the `bind` plugin (if someone tries to install both of them), which can be confusing to debug when it happens.

Furthermore, setting a NAT forward for DNS would be problematic as usually (and to my knowledge) the OS DNS resolvers doesn't support other ports than the default `53`. That said I think changing this to `nat` could cause more potential issues than it really solves, so keep it with `dhcp` after all. Sorry for the confusion.

Reply:
Yes, I agree with you on that matter.
Let`s to it with dhcp

I hope everything is good now and we can apply it to the master branch :D